src/styles/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  background-color: $light-color;
+}
+
+.card__image-container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-color;
+}
+
+.card__product-info {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-color;
+}
```

you need to use scss syntax 
check other places 
```suggestion
.card {
 &__product {
   &-name {
   margin-bottom: 4px;
   font-size: 12px;
   font-weight: 500;
   line-height: 18px;
   color: $main-color;
  }
  
  &-info {
   margin-bottom: 16px;
   font-size: 10px;
   line-height: 14px;
   color: $secondary-color;
  }
 }
}
```